apache/superset
# chore: Select component refactoring - DateFilterControl - Iteration 5

### SUMMARY
It replaces the react-select Select component with the Antdesign one in the DateFilterControl components.

### BEFORE / AFTER

https://user-images.githubusercontent.com/60598000/124779721-4bf33e00-df42-11eb-80bb-9e5ceb8820f0.mp4

https://user-images.githubusercontent.com/60598000/124779858-675e4900-df42-11eb-89d6-c9971beccd70.mp4

### BEFORE / AFTER

https://user-images.githubusercontent.com/60598000/124779875-6c22fd00-df42-11eb-9180-dfc864e493e1.mp4

https://user-images.githubusercontent.com/60598000/124779886-6e855700-df42-11eb-8f13-7943f814e38f.mp4

### TESTING INSTRUCTIONS
1. Open a chart, for instance, the Cross Channel Relationship example chart
2. Click on the "Time" tab 
3. Click on "No filter" under "TIME RANGE"
4. Select a "RANGE TYPE" and observe the behavior
5. Choose the "RANGE TYPE" and set it to "Custom"
6. Set the "START (INCLUSIVE)" and "END (EXCLUSIVE)" options and observe the behavior

### ADDITIONAL INFORMATION
